munkipkg

```diff
@@ -681,6 +799,15 @@ def build(project_dir, options):
         if build_info['distribution_style']:
             build_distribution_pkg(build_info, options)
 
+        # notarize the pkg
+        if "notarization" in build_info and not options.skip_notarization:
+            try:
+                notarize(build_info, options)
```

Comment:
In my (brief) testing I'm seeing it take far longer than 120 seconds for stapling to become available, and of course if there are errors in the notarization process it will never become available. Perhaps we should be running `xcrun altool --notarization-info {output["notarization-upload"]["RequestUUID"]}` periodically waiting for a status .

On status: success proceed to staple, on status: in progress, sleep and try again, on status: invalid (or other error status, print the error (better still, retrieve the log from the LogFileURL) and exit...

I'm not suggesting we get all this in before I'll merge this -- but I think it's needed for a good implementation.

Reply:
I can improve this before the merge. I admit current approach is a bit naive. Even mentioned this problem is PR comment.

Reply:
Check using `altool --notarization-info ` now implemented in 9a6e0d04fa7d518f294cdd649b79f946a99b279d. Default timeout increased to 300 seconds.

Not sure about the retrieving the log from `LogFileURL` though. Currently we display the `Status`, `Status Code`, `Status Message` and `LogFileURL` after a failed notarization attempt. Log can be a very long json. We could download it and put someplace in the `munki-pkg` project directory (Perhaps create `log` directory?).